### ISSUE TITLE
After stopping a node container, selenium hub hangs

### ISSUE THREAD

**Issue by tlkx** [SeleniumHQ/docker-selenium#113]
Hi,

With a selenium hub container running:

```
docker run -d -p 4444:4444 --name selenium-hub selenium/hub
```

If I run a node container (running `docker run -d --link selenium-hub:hub selenium/node-chrome`), it registers to the hub fine, but if I stop the node container, the hub container hangs for a few minutes before responding again.

EDIT: looking at the logs, it seems that the hub tries to reach the node that's down and hangs for the entire duration of this check.

**Comment by alexkogon** [comment]
I believe that is just a bug in the Selenium Grid hub itself...usually this is when you go to its status page, and it actually still works for testing?

**Comment by diemol** [comment]
@tlkx 
I tried to reproduce that with the current images, and it didn't happen. Perhaps it was an issue back in that moment.

Could you please try again with the current images and report back?

Thanks,

**Comment by ddavison** [comment]
@alexkogon is correct, this is typical behavior of the hub, usually due to misconfigured timeouts.  i'd recommend toying around with them if default settings don't work.

**Comment by serialbandicoot** [comment]
Hi @alexkogon, @ddavison, @diemol,

I can still reproduce this issue with Selenium Grid Console v.3.12.0. Would you please be able to elaborate further on changing the timeouts?

thanks...

**Comment by Niceplace** [comment]
@serialbandicoot There's a regression introduced in selenium > 3.8, should be fixed in 3.13­.
If you want more information on how to deal with timeouts, I encourage you to have a look at this issue (https://github.com/SeleniumHQ/selenium/issues/5908) , there is a workaround in there (using SE_OPTS) that I tested not to long ago with 3.12, it works.
